### ISSUE TITLE
False positive for EmptyFunctionBlock with overriden function

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#148]
The following function will be flagged from `EmptyFunctionBlock` with `M12.2`:

```kotlin
class Something : ViewType {
  override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
    // No-op.
  }
}

interface ViewType {
  fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int)
}
```

The compiler won't allow you to remove the `{ }` and hence this should not be flagged.

**Comment by arturbosch** [comment]
Nice catch, I try to fix it asap and release a M12.3 for this.

**Comment by vanniktech** [comment]
Oh wow that was quick 👍

**Comment by arturbosch** [comment]
Just waiting for CI for the new Tag and we are up :)
